### PR TITLE
Update to the latest 6.5 RTM build

### PR DESCRIPTION
**Pull request by mhagen-vmware**

### CODE REVIEW

File: tests/resources/Nimbus-Util.robot [hunk 1]
```diff
@@ -1,6 +1,6 @@
 *** Variables ***
-${ESX_VERSION}  4531672  #6.5 super main
-${VC_VERSION}  4531673   #6.5 super main
+${ESX_VERSION}  4564106  #6.5 RTM
+${VC_VERSION}  4564107   #6.5 RTM
```
Comment (mlh78750) [inline tests/resources/Nimbus-Util.robot]: Latest RTM for VC is 4602587

**Comment by matthewavery** [comment]
LGTM

**Comment by mlh78750** [comment]
LGTM

**Comment by sgairo** [comment]
will change in Nimbus doc  :)

**Comment by rajanashok** [comment]
LGTM

**Pull request merged**